src/useForm.ts

```diff
@@ -117,7 +117,9 @@ export function useForm<
     typeof document !== UNDEFINED &&
     !isWindowUndefined &&
     !isUndefined(window.HTMLElement);
-  const isProxyEnabled = isWeb && 'Proxy' in window;
+  const isProxyEnabled =
+    (isWeb && 'Proxy' in window) ||
+    (typeof navigator !== UNDEFINED && typeof Proxy !== UNDEFINED);
```

Comment:
```suggestion
    isWeb ? 'Proxy' in window : typeof Proxy !== UNDEFINED;
```

hope we can reduce some of the code above, what do you think?

Reply:
sorry been a pain in the a** for writing less code here, I care about the package size a lot, the less we have to write the better it is for me.

Reply:
For me its OK, dont worry

Reply:
Or, its the same and less code
```
const isProxyEnabled = typeof Proxy != 'undefined';
```